osu.Game/Overlays/Volume/MuteButton.cs

```diff
@@ -88,5 +88,11 @@ protected override void OnHoverLost(HoverLostEvent e)
         {
             Content.TransformTo<Container<Drawable>, ColourInfo>("BorderColour", unhoveredColour, 500, Easing.OutQuint);
         }
+
+        protected override bool OnMouseDown(MouseDownEvent e)
+        {
+            base.OnMouseDown(e);
+            return true;
+        }
```

Comment:
I do wonder if this should be done at `OsuButton` to handle other potentially similar failures. Did you happen to try that and come to a reason for not doing so?

Reply:
I did it locally because `OsuButton`s can be used in scroll containers causing drag scroll to not work.

Reply:
Right, that's a scary one. I think an inline comment should be added nothing the reason for adding this.

"Block mouse down to avoid dismissing overlays sitting behind the mute button"